avocado/utils/linux_modules.py

```diff
@@ -40,7 +40,12 @@
 
 
 def load_module(module_name):
-    # Checks if a module has already been loaded
+    """
+    Checks if a module has already been loaded.
+    :param module_name: Name of module to check
+    :return: True if module is loaded, False otherwise
+    :rtype: Bool
```

Comment:
It's not right to combine fixes, docstring updates and improvements in one commit. I can accept it this time, but next time please split those in two separate commits.